yellowbrick/style/colors.py

```diff
@@ -54,58 +54,71 @@ def get_color_cycle():
     return mpl.rcParams['axes.color_cycle']
 
 
-def resolve_colors(num_colors=None, colormap=None, color=None):
+def resolve_colors(n_colors=None, colormap=None, colors=None):
     """
-    Resolves the colormap or the color list with the number of colors.
-    See: https://github.com/pydata/pandas/blob/master/pandas/tools/plotting.py#L163
+    Generates a list of colors based on common color arguments, for example
+    the name of a colormap or palette or another iterable of colors. The list
+    is then truncated (or multiplied) to the specific number of requested
+    colors.
 
     Parameters
     ----------
-    num_colors : int or None
-        the number of colors in the cycle or colormap
-
-    colormap : str or None
-        the colormap used to create the sequence of colors
-
-    color : list or None
-        the list of colors to specifically use with the plot
-
+    n_colors : int, default: None
+        Specify the length of the list of returned colors, which will either
+        truncate or multiple the colors available. If None the length of the
+        colors will not be modified.
+
+    colormap : str, default: None
+        The name of the matplotlib color map with which to generate colors.
+
+    colors : iterable, default: None
+        A collection of colors to use specifically with the plot.
+
+    Returns
+    -------
+    colors : list
+        A list of colors that can be used in matplotlib plots.
+
+    Notes
+    -----
+    This function was originally based on a similar function in the pandas
+    plotting library that has been removed in the new version of the library.
     """
 
-    # Work with the colormap
-    if color is None and colormap is None:
-        if isinstance(colormap, str):
-            cmap = colormap
-            colormap = cm.get_cmap(colormap)
+    # Work with the colormap if specified and colors is not
+    if colormap is not None and colors is None:
```

Comment:
Great, you caught the logic here that was preventing usage of default color scheme.